examples/ios/src/lib.rs

```diff
@@ -87,6 +89,65 @@ fn setup_scene(
         transform: Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
         ..default()
     });
+
+    // Test ui
+    commands.spawn_bundle(UiCameraBundle::default());
```

Comment:
This is causing CI to fail, and needs to be fixed.

Reply:
Done, the example now compile and works on iOS simulator